pure.zsh

```diff
@@ -253,19 +266,22 @@ prompt_pure_async_vcs_info() {
 	# to be used or configured as the user pleases.
 	zstyle ':vcs_info:*' enable git
 	zstyle ':vcs_info:*' use-simple true
-	# Only export three message variables from `vcs_info`.
-	zstyle ':vcs_info:*' max-exports 3
-	# Export branch (%b), Git toplevel (%R), and action (rebase/cherry-pick) (%a).
-	zstyle ':vcs_info:git*' formats '%b' '%R'
-	zstyle ':vcs_info:git*' actionformats '%b' '%R' '%a'
+	# Only export four message variables from `vcs_info`.
+	zstyle ':vcs_info:*' max-exports 4
+	# Export branch (%b), Git toplevel (%R), action (rebase/cherry-pick) (%a),
+	# and stash information via misc (%m).
+	zstyle ':vcs_info:git*' formats '%b' '%R' '%a' '%m'
```

Comment:
I’m guessing there are no repercussions from including `%a` here?

Reply:
For some reason it was producing an error when `formats` didn't have the same amount as `actionformats`. I really don't understand why these two seem to interact, maybe I did something wrong else, still new to zsh internals.

Reply:
Alright, it’s been a while since I worked with vcs_info as well so just being cautious 😄. It’s probably fine.

Reply:
Took a second look... `%m` was only required in `formats` only, but not in `actionformats`.

Reply:
Interesting. How about when a git rebase is active? That’s when action formats kick in AFAICR.

Reply:
And the actionformats are for when git is performing a merge or rebase. If I understand all this correctly, if we were omitting `%m` from actionformats, `info[stash]=$vcs_info_msg_3_` would remain empty (?) and the stash symbol would never be shown during such an action. Which is what we want?

Reply:
I guess we need to ensure that `$vcs_info_msg_3_` would always point to the expected output...